cmd/tk/main.go

```diff
@@ -47,12 +65,12 @@ func main() {
 	)
 
 	// external commands prefixed with "tk-"
-	rootCmd.AddCommand(
+	addCommandsWithLogLevel(
 		prefixCommands("tk-")...,
 	)
 
 	// Run!
 	if err := rootCmd.Execute(); err != nil {
-		log.Fatalln(color.RedString("Error:"), err)
+		fmt.Fprintln(os.Stderr, "Error:", err)
```

Comment:
Seems like this change makes it so that if there is any errors, we don't exit (1).